actions/find.non.existent.floating.ip.projects.yaml

```diff
@@ -0,0 +1,18 @@
+---
+description: Find non-existent projects referenced by floating ips
+enabled: true
+entry_point: src/floating_ip_actions.py
+name: find.non.existent.floating.ip.projects
```

Comment:
These should be grouped into floating_ip operations, i.e. `floating.ip.find.non.existent.projects` or alternatively we have some sort of:
`correctness.find.floating.ip.on.non.existent.project`

(Also I'm getting more and more tempted to move away from the pre-existing `.` separator and switch to the `correctness.find_floating_ip_on_non_existent_project` but not for this PR)